environment.yml

```
@@ -1,60 +1,71 @@
 name: freqtrade
 channels:
-  - defaults
+ # - defaults
   - conda-forge
 dependencies:
-  # Required for app
-  - python>=3.7
-  - pip
-  - wheel
-  - numpy
-  - pandas
-  - SQLAlchemy
-  - arrow
-  - requests
-  - urllib3
-  - wrapt
-  - jsonschema
-  - tabulate
-  - python-rapidjson
-  - flask
-  - python-dotenv
-  - cachetools
-  - python-telegram-bot
-  # Optional for plotting
-  - plotly
-  # Optional for hyperopt
-  - scipy
-  - scikit-optimize
-  - scikit-learn
-  - filelock
-  - joblib
-  # Optional for development
-  - flake8
-  - pytest
-  - pytest-mock
-  - pytest-asyncio
-  - pytest-cov
-  - coveralls
-  - mypy
-  # Useful for jupyter
-  - jupyter
-  - ipykernel
-  - isort
-  - yapf
-  - pip:
-    # Required for app
-    - cython
-    - pycoingecko
-    - ccxt
+# 1/4 req main
+    - python>=3.7
+    - numpy
+    - pandas
+    - pip
+
+    - aiohttp
+    - SQLAlchemy
+    - python-telegram-bot
+    - arrow
+    - cachetools
+    - requests
+    - urllib3
+    - wrapt
+    - jsonschema
     - TA-Lib
-    - py_find_1st
+    - tabulate
+    - jinja2
+    - blosc
     - sdnotify
-    # Optional for develpment
-    - flake8-tidy-imports
-    - flake8-type-annotations
-    - pytest-random-order
-    - -e .
+    - fastapi
+    - uvicorn
+    - pyjwt
+    - colorama
+    - questionary
+    - prompt-toolkit
+
+
+    # ============================
+    # 2/4 req dev
+
+    - coveralls
+    - flake8
+    - mypy
+    - pytest
+    - pytest-asyncio
+    - pytest-cov
+    - pytest-mock=
```

i think the `=` sign is too much here ...

```suggestion
    - pytest-mock
```

---

done, it was in newer update